src/common/WormholeAdapter.sol

```diff
@@ -18,19 +19,19 @@ import {
 contract WormholeAdapter is Auth, IWormholeAdapter {
     using CastLib for bytes32;
 
-    uint16 public immutable refundWormholeId;
+    uint16 public immutable localWormholeId;
     IMessageHandler public immutable gateway;
     IWormholeRelayer public immutable relayer;
 
     mapping(uint16 wormholeId => WormholeSource) public sources;
     mapping(uint16 centrifugeId => WormholeDestination) public destinations;
 
-    constructor(IMessageHandler gateway_, address relayer_, uint16 refundWormholeId_, address deployer)
-        Auth(deployer)
-    {
+    constructor(IMessageHandler gateway_, address relayer_, address deployer) Auth(deployer) {
         gateway = gateway_;
         relayer = IWormholeRelayer(relayer_);
-        refundWormholeId = refundWormholeId_;
+
+        IWormholeDeliveryProvider deliveryProvider = IWormholeDeliveryProvider(relayer.getDefaultDeliveryProvider());
```

Comment:
Suggestion from https://github.com/burrasec/2025-04-Centrifuge-Audit/issues/5

Reply:
Agree!

Reply:
Do you know if we have something similar for Axelar?